src/runtime/entity.ts

```diff
@@ -64,10 +66,11 @@ export interface EntityStaticInterface {
 export type EntityClass = (new (data, userIDComponent?: string) => EntityInterface) & EntityStaticInterface;
 
 export abstract class Entity implements EntityInterface {
-  private userIDComponent?: string;
-
-  private schema: Schema;
   protected rawData: EntityRawData;
+  
+  private _userIDComponent?: string;
+  private _schema: Schema;
+  private _context: ParticleExecutionContext;
```

Comment:
Nit: no need to have the underscore on the names, since we now have the 'private' specifier (although this hasn't been all that consistently applied through the code). Note there are some cases where we've deliberately left underscores on nominally private members, because external code is still directly accessing them and the underscore calls that out.

Reply:
Done. I had to leave _mutable with an underscore, to stop the name collision with the mutable getter and setter functions.

Reply:
Cool. That's actually useful; the underscore indicates something is different for this member.